Nur in util-linux-2.12h-patch/fdisk: cfdisk.c~.
Nur in util-linux-2.12h-patch/fdisk: fdisk.c~.
Nur in util-linux-2.12h-patch/fdisk: sfdisk.c~.
diff -dur util-linux-2.12h/mount/mount.c util-linux-2.12h-patch/mount/mount.c
--- util-linux-2.12h/mount/mount.c	2004-10-13 20:54:39.000000000 +0000
+++ util-linux-2.12h-patch/mount/mount.c	2004-10-23 03:16:30.310508504 +0000
@@ -1109,7 +1109,7 @@
  * Return 0 for success (either mounted sth or -a and NOAUTO was given)
  */
 static int
-mount_one (const char *spec, const char *node, const char *types,
+mount_one (const char *spec, char *node, const char *types,
 	   const char *opts, char *cmdlineopts, int freq, int pass) {
   int status, status2;
   const char *nspec;
Nur in util-linux-2.12h-patch/mount: mount.c~.
Nur in util-linux-2.12h-patch/mount: mount.c.rej.
